nikola/plugins/command/check.py

```diff
@@ -162,13 +174,17 @@ def _execute(self, options, args):
 
     existing_targets = set([])
 
-    def analyze(self, task, find_sources=False):
+    def analyze(self, task, find_sources=False, check_remote=False):
         rv = False
         self.whitelist = [re.compile(x) for x in self.site.config['LINK_CHECK_WHITELIST']]
         base_url = urlparse(self.site.config['BASE_URL'])
         self.existing_targets.add(self.site.config['SITE_URL'])
         self.existing_targets.add(self.site.config['BASE_URL'])
         url_type = self.site.config['URL_TYPE']
+
+        if check_remote and requests is None:
+            utils.req_missing(['requests'], 'check remote links.')
```

Comment:
[grammar] the period should be removed, because the argument is inserted before a comma